README.md

```diff
@@ -47,7 +57,7 @@ Eduardo Ortenblad (edu.ortenblad@mail.utoronto.ca)
 
 Acknowledgements:
 
-This code uses STK
+This code uses STK (https://ccrma.stanford.edu/software/stk/index.html)
 
 This code uses QCustomPlot which is built on QT
```

Comment:
@Maxarreaga can you add links to these pls

Reply:
ok